### ISSUE TITLE
Cannot read eslint config file

### ISSUE THREAD

**Issue by jasonujmaalvis** [SnowdogApps/magento2-frontools#210]
Hello,

I'm facing some issues when trying to run the `gulp eslint --file <name>` task within the `tools` directory under the project root.

It finds the relevant JavaScript file but as soon as I edit the file the task errors saying it cannot read the config file:

`<project>/vendor/snowdog/frontools/config/.eslintrc`

The file doesn't exist in that directory (only the samples), it does exist within:

`<project>/dev/tools/frontools/config`

I'm not sure why its looking in the vendor folder as the `gulp setup` task copies it across and sets up the symlinks.

I'm using the latest version 1.4.5

Just to add to this other configuration files seem to be found ok, e.g BrowerSync is working as expected.

Thanks!

**Comment by Igloczek** [comment]
You can store configs in `vendor/snowdog/frontools/config` - i.e. if you forked the tools and want to keep everything in one place or as you mentioned inside `dev/tools/frontools/config` (it checks this directory first) if you use Frontools with OOTB features and just want to set config files.

Could you show me what kind of modification did you try to make in `.eslintrc`? 
Did you remember to keep the YAML format (2 spaces and proper indentation in the whole file)?

**Comment by jasonalvis** [comment]
Sorry @Igloczek it's user error. Thanks for your help.

**Comment by jasonalvis** [comment]
Hey @Igloczek,

It's reading the configuration file now. I ran `gulp setup` again. However it's reading the wrong config file for eslint. I've tied down the issues to the `configFile` parameter within the `eslint.json` file.

I have the JSON file under the following directory which is being found via gulp correctly:

`dev/tools/frontools/config/eslint.json`

The contents of this file tells it where to load the config from:

```json
{
  "configFile": "./config/.eslintrc"
}
````
For some reason this wasn't working, it would read the config file from `tools/config` instead. 

I've had to remove the `./config/` from the string:

```json
{
  "configFile": ".eslintrc"
}
````

Now it reads the correct file.

**Comment by jasonalvis** [comment]
Hey @Igloczek,

It's reading the configuration file now. I ran `gulp setup` again. 

However after further debugging it's reading the wrong rules file for ESLint. I've tied down the issue to the `configFile` parameter within the `eslint.json` file.

All my config files are held under the following directory:

`dev/tools/frontools/config`

I have the JSON file for ESLint which is being found via gulp correctly:

`dev/tools/frontools/config/eslint.json`

The contents of this file tells it where to load the config from:

```json
{
  "configFile": "./config/.eslintrc"
}
````

As it stands this will load the config file from `tools/config/.eslintrc` but shouldn't it be loading it from `dev/tools/frontools/config/.eslintrc` - where all the other config is - the file exists there but it doesn't appear to ever get read?

To read the config file from that directory I've had to edit the string like so:

```json
{
  "configFile": "../../../dev/tools/frontools/config/.eslintrc"
}
````

Now it reads the correct file. Should this be corrected in the repo?
